UI/GameSettingsScreen.cpp

```diff
@@ -305,7 +305,11 @@ void GameSettingsScreen::CreateViews() {
 #endif
 
 #ifdef _WIN32
-	graphicsSettings->Add(new CheckBox(&g_Config.bVSync, gr->T("VSync")));
+	CheckBox *vsync = graphicsSettings->Add(new CheckBox(&g_Config.bVSync, gr->T("VSync")));
+	vsync->OnClick.Add([=](EventParams &e) {
+		settingInfo_->Show(gr->T("Vsync Tip", "Avoid tearing screen , always on"));
```

Comment:
Always on is probably confusing.  Perhaps, "Avoid screen tearing, depends on video driver"?  What are you trying to communicate here?

-[Unknown]